packages/venia-ui/lib/components/ProductOptions/options.js

```diff
@@ -24,6 +24,8 @@ class Options extends Component {
         const { handleSelectionChange, props } = this;
         const { product, selectedValues = [] } = props;
 
+        // TODO: Do this check in parent and only pass `configurable_options`
+        // instead of the entire `product` as a prop.
```

Comment:
Fun note, we already do this in the mini cart product options page but we don't do it in the PFD. :shrug: